Assets/Scripts/CamControll.cs

```diff
@@ -0,0 +1,28 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CamControll : MonoBehaviour
+{
+    [SerializeField]
+    private Transform target;
+
+    [SerializeField]
+    private Vector3 tOffset;
+
+    [SerializeField]
+    private float cspeed;
```

Comment:
```suggestion
    private float cSpeed;
```

Păstrează convenția (pe care ai și folosit-o mai sus la `tOffset`).